lib/cocoapods/installer/xcode/pods_project_generator/pod_target_integrator.rb

```diff
@@ -0,0 +1,95 @@
+module Pod
+  class Installer
+    class Xcode
+      class PodsProjectGenerator
+        # This class is responsible for integrating a pod target. This includes integrating
+        # the test targets included by each pod target.
+        #
+        class PodTargetIntegrator
+          # @return [PodTarget] the target that should be integrated.
+          #
+          attr_reader :target
+
+          # Init a new PodTargetIntegrator.
+          #
+          # @param  [PodTarget] target @see #target
+          #
+          def initialize(target)
+            @target = target
+          end
+
+          # Integrates the pod target.
+          #
+          # @return [void]
+          #
+          def integrate!
+            UI.section(integration_message) do
+              test_specs, specs = target.specs.partition(&:test_specification?)
+              test_specs.each do |test_spec|
+                native_target = target.native_target_for_spec(test_spec)
+                add_embed_frameworks_script_phase(native_target)
+                add_copy_resources_script_phase(native_target)
+                consumer = test_spec.consumer(target.platform)
+                UserProjectIntegrator::TargetIntegrator.create_or_update_user_script_phases(consumer.script_phases, native_target)
+              end
+              spec_consumers = specs.map { |spec| spec.consumer(target.platform) }
```

Comment:
These two lines are the new addition that integrates shell script phases of podspecs to the native target.